superset/common/query_context.py

```diff
@@ -148,10 +153,10 @@ def cache_timeout(self):
             and self.datasource.database.cache_timeout
         ) is not None:
             return self.datasource.database.cache_timeout
-        return config.get("CACHE_DEFAULT_TIMEOUT")
+        return config["CACHE_DEFAULT_TIMEOUT"]
```

Comment:
Is this guaranteed to exist?

Reply:
I think so - was discussed in earlier comments.